### ISSUE TITLE
Many HTML entities don't work

### ISSUE THREAD

**Issue by ByCzech** [nandoflorestan/kajiki#43]
Many HTML entities i.e. &hellip; cause KeyError:

Try:
`kajiki.XMLTemplate(u'<div>&copy; &hellip;</div>')`

Result:
`Traceback (most recent call last):
  File "<input>", line 1, in <module>
    Template = kajiki.XMLTemplate(u'<div>&copy; &hellip;</div>')
  File "/usr/lib/python2.7/dist-packages/kajiki/xml_template.py", line 52, in XMLTemplate
    doc = _Parser(filename, source).parse()
  File "/usr/lib/python2.7/dist-packages/kajiki/xml_template.py", line 629, in parse
    parser.parse(source)
  File "/usr/lib/python2.7/xml/sax/expatreader.py", line 110, in parse
    xmlreader.IncrementalParser.parse(self, source)
  File "/usr/lib/python2.7/xml/sax/xmlreader.py", line 123, in parse
    self.feed(buffer)
  File "/usr/lib/python2.7/xml/sax/expatreader.py", line 213, in feed
    self._parser.Parse(data, isFinal)
  File "/usr/lib/python2.7/xml/sax/expatreader.py", line 416, in skipped_entity_handler
    self._cont_handler.skippedEntity(name)
  File "/usr/lib/python2.7/dist-packages/kajiki/xml_template.py", line 675, in skippedEntity
    return self.characters(html5[name])
KeyError: u'hellip'`

**Comment by philippkraft** [comment]
Same is true for greek letters and math symbols

**Comment by philippkraft** [comment]
The problem seems to be, that the entity translator dict `kajiki.entities.html5` does contain some keys with and without a trailing semicolon, while most entities are only saved with a trailing semicolon. The code referenced in the traceback of @ByCzech shows the key without a trailing semicolon. A cheap fix would be to extend the dict with all keys that have a semicolon with a copy of the value without the semicolon. 

The following monkey patch works for me:

```.py
import kajiki
kajiki.entities.html5.update(
    {k[:-1]: v 
     for k, v in kajiki.entities.html5.items() 
     if k[-1]==';' and k[:-1] not in kajiki.entities.html5
})

from kajiki import XMLTemplate
t = XMLTemplate('<xml>&hellip;</xml>')
print(t().render())
```

If this fast fix is ok for now, I can provide a PR, of course.

**Comment by amol-** [comment]
https://github.com/nandoflorestan/kajiki/commit/35e916d7e60b7224a364aca3316556c23ba3ef1c

should provide a reasonable solution to the problem without having to maintain our own delta from https://github.com/python/cpython/blob/master/Lib/html/entities.py#L264

**Comment by philippkraft** [comment]
Thank you for the fast solution, for now I am going with this, do you think the version estimate to get it fixed in a PyPI release ok? 

```.py
if kajiki.__version__ < '0.9':
    kajiki.entities.html5.update(
        {k[:-1]: v
         for k, v in kajiki.entities.html5.items()
         if k[-1] == ';' and k[:-1] not in kajiki.entities.html5
    })

```

**Comment by amol-** [comment]
Released 0.8.2 https://pypi.org/project/Kajiki/0.8.2/
